src/fairchem/core/preprocessing/atoms_to_graphs.py

```diff
@@ -77,6 +77,8 @@ class AtomsToGraphs:
         Default is False, so the periodic boundary conditions will not be returned.
         r_data_keys (sequence of str, optional): Return values corresponding to given keys in atoms.info data with other
         properties. Default is None, so no data will be returned as properties.
+        molecule_cell_size: if the atom object doesn't have a cell (its volume is 0.0), then create a large molecular box
+        and the center the atoms in the middle, units are Angstroms
```

Comment:
add comment that, you can be generous with the padding because it makes no difference on the speed of pymatgen neighbor search